pkg/noderesourcetopology/filter.go

```diff
@@ -24,22 +24,18 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
 	v1qos "k8s.io/kubernetes/pkg/apis/core/v1/helper/qos"
 	bm "k8s.io/kubernetes/pkg/kubelet/cm/topologymanager/bitmask"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 	apiconfig "sigs.k8s.io/scheduler-plugins/pkg/apis/config"
 
 	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
-	topoclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
-	topologyinformers "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/informers/externalversions"
-	listerv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/listers/topology/v1alpha1"
 )
 
 const (
-	// Name is the name of the plugin used in the plugin registry and configurations.
-	Name = "NodeResourceTopologyMatch"
+	// FilterPluginName is the name of the plugin used in the plugin registry and configurations.
+	FilterPluginName = "NodeResourceTopologyMatch"
```

Comment:
Probably there is no strictly defined coding convention, but most plugins have const Name which keeps plugin name, it could be not so convinient (for  readability purpose only) if we have many places where it used.

Reply:
I agree, but now we have 2 plugins on the same package, we have the Filter plugin and the Score plugin, so we can't set both of them under ```Name```. 
What I can do is separate it into three different packages:
```filter```
```score```
and ```plugin_helpers``` which will be consumed by both packages. What do you think?

Reply:
Separating into the packages as suggested above sounds okay to me!